src/components/TodoList/TodoList.jsx

```diff
@@ -1,4 +1,9 @@
-// Add the required props
-export const TodoList = () => (
-  <>TodoList markup</>
+import { TodoInfo } from '../TodoInfo';
+
+export const TodoList = ({ todos }) => (
+  <section className="TodoList">
+    {todos.map(todo => (
+      <li><TodoInfo key={todo.id} todo={todo} /></li>
+    ))}
+  </section>
```

Comment:
```suggestion
  <ul className="TodoList">
    {todos.map(todo => (
      <TodoInfo key={todo.id} todo={todo} />
    ))}
  </ul>
```
use `li` in TodoInfo instead `article`
and `li` shoul be in `ul` or ol tag (for this case `ul`).
also reset default styles for list
set `list-style: none` for .TodoList, also ul has other default styles which should be resetted (margin, padding...)